### ISSUE TITLE
Reconsider 30 days trash bin automatic deletion period

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/calendar#6213]
### Is your feature request related to a problem? Please describe.

Currently we delete calendar events automatically after 30 days, which is a bit on the harsh side. We don’t do that even in Files, where content is potentially much bigger.

### Describe the solution you'd like

For Files we do some nice automatism where we don’t purge things if there is still space in the quota, so we could do it similarly for Calendar.

### Describe alternatives you've considered

As a short-term solution we could simply up the limit to 12 months? That would take care of a lot of deletions, also by others.

### Additional context

An event was deleted by accident a while ago and is not recoverable:
![Screenshot_20240731_143855](https://github.com/user-attachments/assets/34fc9603-673b-42b4-890b-c0be2fe7d373)

**Comment by section1** [comment]
@jancborchardt i think you can change this value with occ... never try it ..docs:

https://docs.nextcloud.com/server/latest/admin_manual/groupware/calendar.html#trash-bin

**Comment by st3iny** [comment]
Yeah, it can be adjusted. However, Jan is talking about a more sane default value for all instances which makes sense.
